src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -488,6 +488,107 @@ akka {
       }
     }
 
+    networkstream {
+      # The class given here must implement the akka.remote.transport.Transport
+      # interface and offer a public constructor which takes two arguments:
+      #  1) akka.actor.ExtendedActorSystem
+      #  2) com.typesafe.config.Config
+      transport-class = "Akka.Remote.Transport.Streaming.NetworkStreamTransport, Akka.Remote"
+
+      # Transport drivers can be augmented with adapters by adding their
+      # name to the applied-adapters list. The last adapter in the
+      # list is the adapter immediately above the driver, while
+      # the first one is the top of the stack below the standard
+      # Akka protocol
+      applied-adapters = []
+
+      # The default remote server port clients should connect to.
+      # Default is 2552 (AKKA), use 0 if you want a random available port
+      # This port needs to be unique for each actor system on the same machine.
+      port = 2552
+
+      # The hostname or ip clients should connect to.
+      hostname = "localhost"
+
+      # Use this setting to bind a network interface to a different port
+      # than remoting protocol expects messages at. This may be used
+      # when running akka nodes in a separated networks (under NATs or docker containers).
+      # Use 0 if you want a random available port. Examples:
+      #
+      # akka.remote.networkstream.port = 2552
+      # akka.remote.networkstream.bind-port = 2553
+      # Network interface will be bound to the 2553 port, but remoting protocol will
+      # expect messages sent to port 2552.
+      #
+      # akka.remote.networkstream.port = 0
+      # akka.remote.networkstream.bind-port = 0
+      # Network interface will be bound to a random port, and remoting protocol will
+      # expect messages sent to the bound port.
+      #
+      # akka.remote.networkstream.port = 2552
+      # akka.remote.networkstream.bind-port = 0
+      # Network interface will be bound to a random port, but remoting protocol will
+      # expect messages sent to port 2552.
+      #
+      # akka.remote.networkstream.port = 0
+      # akka.remote.networkstream.bind-port = 2553
+      # Network interface will be bound to the 2553 port, and remoting protocol will
+      # expect messages sent to the bound port.
+      #
+      # akka.remote.networkstream.port = 2552
+      # akka.remote.networkstream.bind-port = ""
+      # Network interface will be bound to the 2552 port, and remoting protocol will
+      # expect messages sent to the bound port.
+      #
+      # akka.remote.networkstream.port if empty
+      bind-port = ""
+
+      # Use this setting to bind to a specific ip
+      # than remoting protocol expects messages at.
+      # Use "0.0.0.0" to bind to all interfaces.
+      # akka.remote.networkstream.hostname if empty
+      bind-hostname = ""
```

Comment:
nitpick: we use "public-hostname" to achieve this same thing on the existing Helios transport. I don't have a strong opinion on whether or not those should be consistent.